client/app/account/billing/main/payments/details/billing-payments-details.routes.js

```diff
@@ -0,0 +1,10 @@
+angular.module("Billing").config(($stateProvider, BILLING_BASE_URL) => {
+
+    $stateProvider.state("app.account.billing.main.paymentsDetails", {
+        url: "/payments/:id/details",
+        templateUrl: `${BILLING_BASE_URL}/main/payments/details/billing-payments-details.html`,
```

Comment:
you can remove `BILLING_BASE_URL` and use `account/billing/` instead as it is now the same for both environment (dev and prod)